indico/modules/events/sessions/controllers/management/sessions.py

```diff
@@ -119,3 +126,51 @@ def _process_PATCH(self):
             updates['colors'] = colors
         update_session(self.session, updates)
         return jsonify()
+
+
+class RHSessionPersonList(RHManageSessionsBase):
+    """List of persons in the session"""
+
+    def _process(self):
+        session_ids = map(int, request.form.getlist('session_id'))
+        session_persons = (ContributionPersonLink
+                           .find(ContributionPersonLink.person_id == EventPerson.id,
+                                 Contribution.session_id.in_(session_ids))
+                           .join(ContributionPersonLink.contribution).all())
+        session_persons.extend(SubContributionPersonLink
+                               .find(SubContributionPersonLink.person_id == EventPerson.id,
+                                     Contribution.session_id.in_(session_ids))
+                               .join(SubContributionPersonLink.subcontribution)
+                               .join(SubContribution.contribution))
+
+        return jsonify_template('events/sessions/management/session_person_list.html',
+                                session_persons=session_persons, event=self.event_new)
+
+
+class RHSessionsEmailPersons(RHManageSessionsBase):
+    """Send emails to selected EventPersons"""
+
+    def _checkParams(self, params):
+        self._doNotSanitizeFields.append('from_address')
+        RHManageSessionsBase._checkParams(self, params)
+
+    def _create_form(self):
+        return EmailSessionPersonsForm(event=self.event_new)
+
+    def _process_GET(self):
```

Comment:
Again, for form-based flows having separate methods for GET and POST is not a good idea. You want to show the form again in case validation fails (which, by the way, you don't perform at all).